Civi/Api4/System.php

```diff
@@ -57,13 +57,4 @@ public static function getFields($checkPermissions = TRUE) {
     }))->setCheckPermissions($checkPermissions);
   }
 
-  /**
-   * @param bool $checkPermissions
-   * @return Action\System\ResetPaths
-   */
-  public static function resetPaths($checkPermissions = TRUE) {
```

Comment:
While I agree that it seems unlikely anyone is using this, maybe should deprecate it for a bit first?

Reply:
Ok if we must... :innocent:

Reply:
I think we should add the @deprecated tag to the class, so that it shows up in api explorer as deprecated, and then instead of a log warning it should use \CRM_Core_Error::deprecatedWarning(). That way it's more visible and in the crazy event someone is using it in a unit test it will cause a test fail.

Also did you want to add the civicrm_generated file - not a blocker we can do that later.